pkg/asset/machines/clusterapi.go

```diff
@@ -301,10 +304,34 @@ func (c *ClusterAPI) Generate(dependencies asset.Parents) error {
 		mpool.Set(pool.Platform.VSphere)
 
 		platform := ic.VSphere
+		resolver := &net.Resolver{
+			PreferGo: true,
+		}
 
 		for _, v := range platform.VCenters {
-			err := installConfig.VSphere.Networks(ctx, v, platform.FailureDomains)
+			// Defense against potential issues with assisted installer
+			// If the installer is unable to resolve vCenter there is a good possibility
+			// that the installer's install-config has been provided with bogus values.
```

Comment:
Can't this be validated during install-config validation then?

Reply:
I suppose it could be in `ValidatePlatform` but do you have a suggestion how I would ignore errors here?

Reply:
Oh, so you can't error out during install-config/platform validation? Then I guess this is fine as it is.